packages/conf-haveged/conf-havaged.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+homepage: "https://linux.die.net/man/8/haveged"
+author: "A. Seznec and N. Sendrier"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-1.0-or-later"
+depexts: [
+# ["havaged"] {os = "openbsd"} # not in openbsd?!
+# ["havaged"] {os-family = "debian"} # not in Debian?!
```

Comment:
```suggestion
  ["havaged"] {os-family = "debian"}
```
It seems to exist https://packages.debian.org/buster/haveged

Reply:
ok, thanks